File: problem_fortheen.js
// how many singal caraacter have this sentence
var value ="hello payer bhai"
function find_character(value){
    var val = value.split(/\s/).join('')
    var lst = val.split("")
    var dic ={}
    for (var i =0 ; i<lst.length; i++){
        if (lst[i] in dic){
            dic[lst[i]]+=1
        }
        else{
            dic[lst[i]] =1
        }
    }
    return dic
}
console.log(find_character(value))

File: problem_fortheen.py
# how many singal  caraacter have this sentence.
value= input("Enter a sentence:")
def Myfunction(data):
    val = data.replace(" ", "")
    lst = list(val)
    dic={}
    for i in lst:
        if i in dic:
            dic[i]+=1
        else:
            dic[i]= 1
    return dic
print(Myfunction(value))